### PR TITLE
Add December 2023, January 2024

### PR DESCRIPTION
**Pull request by Bestem0r**
- Adds blog post for what has been done during December 2023 and January 2024

### CODE REVIEW

File: _posts/2024-02-01-december-january.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+![Konfetti](/images/posts/2024-02-01-konfetti.png)
+
+# Grupper i sidemenyen
+
+Endeleg kan du sjå alle abakus sine grupper i sidemenyen! 🤩 Dette inkluderer både styrer, grupper og komitear!
+
+| Før                                                                                                           | Etter                                                                                                           |
+|---------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| ![Før](https://user-images.githubusercontent.com/13599770/297894667-1e4d07d0-5d17-40b8-8202-898adeaad0e9.png) | ![Etter](https://user-images.githubusercontent.com/13599770/297894715-7d6ed3c3-157a-44de-b5bc-7aa8bca95072.png) |
```
Comment (haukkagu) [inline _posts/2024-02-01-december-january.md:21]: Dette har vel vært her en stund. Det som er nytt er vel at de er synlige selv for de som ikke er logget inn?

**Review by haukkagu — APPROVED** [review]
Nice!

File: _posts/2024-02-01-december-january.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+# Konfetti når ein opprettar ein brukar
+
+For å feire nye abakule, har me no lagt til konfetti når ein opprettar ein brukar! 🎊 Litt ekstra stas altså! 🥳
+
+![Konfetti](/images/posts/2024-02-01-konfetti.png)
+
+# Grupper i sidemenyen
+
+Endeleg kan du sjå alle abakus sine grupper i sidemenyen! 🤩 Dette inkluderer både styrer, grupper og komitear!
```
Comment (ivarnakken) [inline _posts/2024-02-01-december-january.md:17]: Ideally write Abakus with a capital A, but no worries, no need to fix

**Review by ivarnakken — APPROVED** [review]
So when will these be written in Norwegian

File: _posts/2024-02-01-december-january.md (hunk 1 above)
Comment (Bestem0r) [reply _posts/2024-02-01-december-january.md:21]: Ah right. Removing the section then, since it's not really news

**Pull request merged**